modern/jsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "baseUrl": "src"
+    "baseUrl": "./"
```

Comment:
Why do we need changes in this file?

Reply:
On the [SocketController lines 3](https://github.com/traccar/traccar-web/blob/4d200181a6a97c8143d0843f07d90072ef35e25c/modern/src/SocketController.js#L3) or [DeviceList lines 14](https://github.com/traccar/traccar-web/blob/4d200181a6a97c8143d0843f07d90072ef35e25c/modern/src/DeviceList.js#L14)
 
You can see that I am importing from **store** using `import ... from 'src/store';`. If I use `"baseUrl": "src"`, I could import it like: `import ... from 'store';`

But I'm afraid it would create a conflict with some library or nodejs modules, So I think it better to use `import ... from 'src/store';`

Reply:
Why do we need it in the first place?

Reply:
If, later, the project become more complex with so many subfolders. We would not have to import like this: `import ... from '../../../store'; or `import ... from '../../../SomeComponent'`

Reply:
That makes sense, but:
1. Let's split that into a separate pull request
2. If we do it, let's update all imports

Reply:
OK then, I will remove it.